scripts/update

```diff
@@ -7,64 +7,66 @@ fi
 
 DIR="$(dirname "$0")"
 
-
 function usage() {
-
     echo -n \
 "Usage: $(basename "$0")
-Setup external project dependencies.
+
+Update project dependencies, build assembly JARs, run database migrations, 
+and build static asset bundle.
 "
 }
 
 function run_database_migrations() {
     # Check if database migrations have already been initialized
     set +e
     docker-compose \
-        exec -T postgres gosu postgres psql -d rasterfoundry -c 'select 1 from __migrations__' &>/dev/null
+        exec -T postgres \
+        psql -U rasterfoundry -d rasterfoundry -c 'select 1 from __migrations__' &>/dev/null
     status_check=$?
     set -e
-    if [ $status_check == 0 ]
-    then
+    if [ $status_check == 0 ]; then
         echo "Migrations already initialized"
     else
         # Initialize the database for migrations.
         docker-compose \
-            run --rm api-server "mg init"
+            run --rm sbt \
+            "migrations/run init"
     fi
 
     # Run database migrations. The way scala-forklift works requires this to be called twice:
     # the first run figures out the migrations to run and the second run applies them.
     docker-compose \
-        run --rm api-server "mg update"
-    docker-compose \
-        run --rm api-server "mg apply"
+        run --rm sbt \
+        ";migrations/run update;migrations/run apply"
 }
 
-
-if [ "${BASH_SOURCE[0]}" = "${0}" ]
-then
-    if [ "${1:-}" = "--help" ]
-    then
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
         usage
     else
         echo "Updating Scala dependencies"
-        docker-compose up -d api-server
+        docker-compose \
+            run --rm --no-deps sbt \
+            update
 
-        docker-compose exec -T api-server ./sbt update
-        docker-compose rm -sf api-server
+        echo "Building Scala assembly JARs"
+        docker-compose \
```

Comment:
Since we're running the assembly JARs in `server`, they should be created in `update` as part of bootstrapping the development environment, etc.